packages/icons/src/icon/stories/keywords.ts

```diff
@@ -0,0 +1,13 @@
+const keywords: Partial< Record< keyof typeof import('../../'), string[] > > = {
+	cancelCircleFilled: [ 'close' ],
+	create: [ 'add' ],
+	file: [ 'folder' ],
+	seen: [ 'show' ],
+	thumbsDown: [ 'dislike' ],
+	thumbsUp: [ 'like' ],
+	trash: [ 'delete' ],
+	unseen: [ 'hide' ],
+	warning: [ 'alert', 'caution' ],
```

Comment:
@WordPress/gutenberg-design I assume there will also be some keywords we _don't_ want to add, to discourage people misusing icons for certain things? I added some obvious keywords to start, but let me know if there's something that should be removed. (Maybe we could document those no-nos.)